website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,111 @@
+- Username/Password
+- [OIDC](/boundary/tutorials/identity-management/oidc-auth)
+- LDAP
+
+### Accounts
+[Accounts](/boundary/docs/concepts/domain-model/accounts) are resources that represent an authenticated user from a trusted identity provider through a Boundary auth method. Accounts must belong to an auth method, and can be associated with a single Boundary User. Boundary accounts can belong to the global or Org Scopes.
+
+### Users
+[Users](/boundary/docs/concepts/domain-model/users) are resources that represent an individual person or entity for the purposes of
+access control. A user can contain multiple accounts from multiple auth methods, and can belong to the global or Org Scopes.
+
+### Groups
+[Groups](/boundary/docs/concepts/domain-model/groups) are resources that represent a collection of users, and can be contained by any
+scope. [Managed groups](/boundary/docs/concepts/domain-model/managed-groups) are a special kind of group that syncs with the auth
+method's identity provider to provide up-to-date information.
+
+### Roles
+[Roles](/boundary/docs/concepts/domain-model/roles) are resources that contain a collection of
+[permissions](/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles
+can be contained by any scope, and the permissions can be applied to the same scope or any child scope.
+
+## Access management
+### Configuring users - username/password
+There are 3 steps to adding new users to Boundary using the password auth method.
+1. Choose the **scope** and the **auth method** that the user will use to authenticate to Boundary. If the desired auth method does not exist, a new one needs
+to be created.
+2. After the auth method is selected, an **account** can be created under the auth method.
+3. In the same scope, create a new **user**, and then attach the new account to the new user. The user is now able to authenticate to Boundary.
+
+### Configuring users - OIDC/LDAP
+[OIDC](/boundary/tutorials/identity-management/oidc-auth) and LDAP accounts and users are auto-vivified (automatically generated) in Boundary in the same scope
+as the auth method. The accounts and users are only created once the user authenticates for the first time to Boundary. The same applies to OIDC/LDAP
+[managed groups](/boundary/tutorials/identity-management/oidc-idp-groups).
+
+### Granting permissions
+When setting up access controls for a user, it is important to first consider which scope(s) the user needs access to. **Each scope requires its own set of roles**,
+giving users permission to perform actions through [grants strings](/boundary/docs/concepts/security/permissions/grants).
+It is recommended to add the user to a group, and then add the group to the role(s), instead of adding the user directly to the role(s). This way, multiple users can
+be managed at the same time, and it is easier to
```

Comment:
```suggestion
It is recommended to add the user to a group, and then add the group to the role(s), instead of adding the user directly to the role(s). This way, you can manage multiple users at the same time, and it is easier to
```